app/scripts/modules/core/src/managed/ArtifactDetail.tsx

```diff
@@ -147,7 +149,15 @@ const EnvironmentCards = memo(
         replacedAt={replacedAt}
         replacedBy={replacedBy}
         vetoed={vetoed}
+        compareLink={compareLink}
         allVersions={allVersions}
+        logClick={(message) => {
+          ReactGA.event({
+            category: 'Environments - version details',
+            action: message,
+            label: `${application.name}:${environmentName}:${reference}`,
+          });
```

Comment:
@gal-yardeni for now, this is how we'll be able to track usage of this button. I'm also logging the app name, environment name, and artifact reference name.

Reply:
@erikmunson how will this information be surfaced?

Reply:
For us at Netflix it will get pumped into our Google Analytics workspace where we can query it - this is how we do analytics across all of deck today. I am looking into more full-featured product analytics options but for now GA should get us what we need.